.github/workflows/tests.yml

```diff
@@ -59,3 +61,21 @@ jobs:
                   path: |
                       coverage
                       !coverage/lcov-report
+
+    matrix-react-sdk:
+        name: Downstream test matrix-react-sdk
+        if: github.event_name == 'merge_group'
+        uses: matrix-org/matrix-react-sdk/.github/workflows/tests.yml@develop
```

Comment:
Could we run this workflow without any of the coverage option? 
That will significantly speed up test execution, and we don't need to check for coverage in this scenario

Reply:
Done, requires https://github.com/matrix-org/matrix-react-sdk/pull/10198